blackbox/docs/general/user-defined-functions.rst

```diff
@@ -190,7 +190,8 @@ Optionally, you can provide a schema::
 Supported Languages
 ===================
 
-CrateDB currently only supports the UDF language ``javascript``.
+CrateDB currently only supports the UDF language ``javascript``, which needs to 
+be turned on as :ref:`CrateDB setting <conf-node-lang-js>`.
```

Comment:
Does this note further below (https://github.com/crate/crate/pull/8237/files#diff-7961a35d46e278834a6507c7780e693eR223) still make sense if this is up here?